docs(validation): add backend validator example

MathieuPuech opened the pull request:
## What I did

1. Add a backend validator example

jorenbroekema commented on packages/form-integrations/docs/17-validation-examples.md:
```diff
@@ -695,3 +695,80 @@ export const FormValidationReset = () => html`
+          .validators="${[new BackendValidator(''), new Required()]}"
+          .modelValue="${''}"
+        ></lion-input>
+        <lion-checkbox-group .mulipleChoice="${false}" name="simulateError">
```
```suggestion
        <lion-checkbox-group .multipleChoice="${false}" name="simulateError">
```

jorenbroekema reviewed (approved):
I like this approach! nice. Comment is just a small nitpick, perhaps multipleChoice prop binding can be left out entirely?

MathieuPuech commented:
> I like this approach! nice. Comment is just a small nitpick, perhaps multipleChoice prop binding can be left out entirely?

yes, I removed it completely and added some improvements

jorenbroekema commented on packages/form-integrations/docs/17-validation-examples.md:
```diff
@@ -695,3 +695,92 @@ export const FormValidationReset = () => html`
+  }
+  return html`
+    <style>
+      lion-input[is-pending] {
```
Where does is-pending attribute come from?

MathieuPuech replied:
It's added by the `async` validators

https://github.com/ing-bank/lion/blob/ee85fa036e3a7f79e93ea391fec181657833eadb/packages/form-core/src/validate/ValidateMixin.js#L71

jorenbroekema replied:
Oooh cool, didn't know!

jorenbroekema commented:
Looks good to me! Feel free to merge whenever
